cupy/linalg/__init__.py

```diff
@@ -37,6 +37,7 @@
 from cupy.linalg._solve import inv  # NOQA
 from cupy.linalg._solve import pinv  # NOQA
 from cupy.linalg._solve import tensorinv  # NOQA
+from cupy.linalg._solve import cho_solve
```

Comment:
`cho_solve` should live in `cupyx.scipy.linalg` since there is no NumPy counterpart